providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -1421,3 +1425,10 @@ def __exit__(self, exctype, excinst, exctb) -> bool:
             logger = logging.getLogger(__name__)
             logger.exception(excinst)
         return True
+
+# --- Helper functions ---
+def normalize_labels_dict(labels: dict) -> dict:
+    """
+    Return a copy of the labels dict with all None values replaced by empty strings.
+    """
+    return {k: ('' if v is None else v) for k, v in labels.items()}
```

Comment:
```suggestion
    return {k: ("" if v is None else v) for k, v in labels.items()}
```

To fix static checks